include/aws/common/array_list.inl

```diff
@@ -404,15 +392,6 @@ int aws_array_list_set_at(struct aws_array_list *AWS_RESTRICT list, const void *
     return AWS_OP_SUCCESS;
 }
 
-AWS_STATIC_IMPL
-void aws_array_list_sort(struct aws_array_list *AWS_RESTRICT list, aws_array_list_comparator_fn *compare_fn) {
-    AWS_PRECONDITION(aws_array_list_is_valid(list));
-    if (list->data) {
-        qsort(list->data, aws_array_list_length(list), list->item_size, compare_fn);
```

Comment:
moved this implementation to another file: `.inl` -> `.c`
and changed it so it wasn't static/inline anymore
it was causing warnings because it called qsort() and couldn't be inlined